src/applications/disability-benefits/526EZ/components/IntroductionPage.jsx

```diff
@@ -60,12 +56,7 @@ class IntroductionPage extends React.Component {
           Related Compensation Benefits).
         </p>
         <BetaGate>
-          <FormStartControls
```

Comment:
As I understand it, it appears there used to be some UI for in-progress forms here. The designs did not appear to accommodate this feature, so I'm wondering if this will have to be implemented again after CTAs are complete.